### DISCUSSION TITLE
Remark42 does not work properly on the EJS template

### DISCUSSION THREAD

**Discussion by nvkha** [umputun/remark42#1628] [Q&A]
Sorry for my bad English. I have an issue when I place remark42 on my EJS template page, after I log in with the social media method, nothing happens (change UI, display "Send" button). I'm pretty sure it's successfully logged but I need to reload the page to see the "Send" button and begin the comment. I already test on the HTML page and it's working fine but on EJS it's not working. Please help me, thanks.

**Comment by umputun** [comment]
pls use [Discussions](https://github.com/umputun/remark42/discussions) for questions like this. Transfered

**Reply by nvkha** [reply]
Sorry for my mistake, thank you

**Comment by paskal** [comment]
Did you manage to solve your problem?
